.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,31 +4,52 @@ about: Create a bug report to help us improve
 labels: bug, needs triage
 ---
 
-<!---
+<!--
 When creating a bug report please:
-- Verify first that your issue is not already reported on GitHub
+- Verify first that your issue is not already reported on GitHub.
 - Test if the latest release and master branch are affected too.
-- Provide a clear and concise description of what the bug is in "Bug report 
-  summary" section.
-- Try to provide as much information about your environment (OS distribution,
-  running in container, etc.) as possible to allow us reproduce this bug faster.
-- Write which component is affected. We group our components the same way our
-  code is structured so basically:
-  component name = dir in top level directory of repository
-- Describe how you found this bug and how we can reproduce it. Preferable with
-  a minimal test-case scenario. You can paste gist.github.com links for larger
-  files
-- Provide a clear and concise description of what you expected to happen.
 -->
 
 ##### Bug report summary
+<!-- Provide a clear and concise description of what the bug. -->
 
-##### OS / Environment (output of `uname -a; command -v sw_vers && sw_vers || grep -Hv "^#" /etc/*release`)
+##### OS / Environment
+<!--
+Provide as much information about your environment (OS distribution, running in container, etc.)
+as possible to allow us reproduce this bug faster.
 
-##### Netdata version (ouput of `netdata -V`)
+To get this information execute:
+- uname -a; grep -Hv "^#" /etc/*release  # linux/bsd
+- uname -a; sw_vers                      # macOS
+
```

Comment:
The previous 3 lines should be on `OS /Environment`

Reply:
Sorry, I only saw now that you are removing Netdata version.

Reply:
i havent remove anything, i moved stuff to appropriate section.

> The previous 3 lines should be on OS /Environment

Those limes under `OS /Environment` section.